content/applications/inventory_and_mrp/manufacturing/management/work_center_time_off.rst

```diff
@@ -0,0 +1,121 @@
+============================================
+Make work centers unavailable using Time Off
+============================================
+
+In Odoo, *work centers* are used to carry out manufacturing operations at specific locations.
+However, if a work center cannot be used for some reason, work orders begin to pile up at the work
+center until it is operational again.
+
+As a result, it is necessary to make the work center unavailable in Odoo so that the platform routes
+new work orders to alternative work centers that are operational. Using Odoo :guilabel:`Time Off`,
+it is possible to designate a work center as being unavailable for a set period of time. Doing
+so ensures that manufacturing operations can continue until the impacted work center is available
+again.
+
+Configuration
+=============
+
+Before a work center can be designated as unavailable, the Odoo platform must be properly
+configured. First, it is necessary to enable developer mode. This allows the :guilabel:`Time Off`
+smart button to appear on each work center's :guilabel:`Working Hours` pop-up window.
+
+Enable developer mode by going to :guilabel:`Settings`, scrolling to the bottom of the page, and
+clicking :guilabel:`Activate the developer mode` under the **Developer Tools** heading.
+
+.. image:: work_center_time_off/developer-mode-button.png
+   :align: center
+   :alt: The "Activate the developer mode" button.
+
+Next, install the :guilabel:`Time Off` app. This is the app used for assigning time off to all
+resources within Odoo, including employees and work centers. Navigate to :menuselection:`Apps`, then
+type "Time Off" in the :guilabel:`Search...` bar. The card for the :guilabel:`Time Off` module
+should be the only one that appears on the page. Click the green :guilabel:`Install` button on the
+card to install the app.
```

Comment:
Active instruction hasnt started yet, so italicize _Time Off_, early line break "Type", and user input should be in literal (backticks)
```suggestion
Next, install the *Time Off* app. This is the app used for assigning time off to all resources
within Odoo, including employees and work centers. Navigate to :menuselection:`Apps`, then type
`Time Off` in the :guilabel:`Search...` bar. The card for the :guilabel:`Time Off` module should be
the only one that appears on the page. Click the green :guilabel:`Install` button on the card to
install the app.
```